lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -115,8 +115,8 @@ defmodule ExUnit.CaptureLog do
           {:DOWN, ^ref, :process, ^parent, _reason} -> :ok
           {:gen_event_EXIT, {Console, ^pid}, _reason} -> :ok
         end
-      other ->
-        :proc_lib.init_ack(other)
+      {:error, exit_reason} ->
```

Comment:
Can you verify that these are the only two possible return values?

Reply:
OK! I'll check and get back to you...

Reply:
You're right, [according to the specs](http://erlang.org/doc/man/gen_event.html#add_sup_handler-3), it returns `:ok`, `{:EXIT, reason}` or any `term`. I will make some more tests and fix this.

Reply:
Just some more information about `gen_event:add_sup_handler/3`:

> If `Module:init/1` returns a correct value indicating successful completion, the event manager adds the event handler and this function returns `ok`. If `Module:init/1` fails with `Reason` or returns `{error,Reason}`, the event handler is ignored and this function returns `{'EXIT',Reason}` or `{error,Reason}`, respectively.

Extracted from [`gen_event:add_handler/3` docs](http://erlang.org/doc/man/gen_event.html#add_handler-3).